acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuBeaconNode.java

```diff
@@ -484,6 +485,19 @@ public void waitForNonDefaultExecutionPayload() {
         MINUTES);
   }
 
+  public void waitForBlockSatisfying(final ThrowingConsumer<? super SignedBlock> consumer) {
+    LOG.debug("Wait for a block satisfying certain assertions");
+
+    waitFor(
+        () -> {
+          final Optional<SignedBlock> block = fetchHeadBlock();
+          assertThat(block).isPresent();
+          assertThat(block.get()).satisfies(consumer);
+        },
+        5,
+        MINUTES);
```

Comment:
5 minutes is a long timeout for a default...

Reply:
copy-paste
will fix